idea/idea-fir/src/org/jetbrains/kotlin/idea/quickfix/MainKtQuickFixRegistrar.kt

```diff
@@ -63,9 +64,14 @@ class MainKtQuickFixRegistrar : KtQuickFixRegistrar() {
         registerPsiQuickFixes(KtFirDiagnostic.ValWithSetter::class, ChangeVariableMutabilityFix.VAL_WITH_SETTER_FACTORY)
     }
 
+    private val expressions = KtQuickFixesListBuilder.registerPsiQuickFix {
```

Comment:
Do we have any conventions or guidelines for this? How about matching with FIR diagnostic group, `nullability`?

Reply:
There is a many-to-many mapping from checker to fixes, and the categories don't always fall neatly together. I think the convention thus far is to group fixes based on what they operate on, hence `expressions`.